### PR TITLE
deps: upgrade to rustwide 0.4.0

### PR DESCRIPTION
**Pull request by pietroalbini**
r? @jyn514

### CODE REVIEW

File: Cargo.lock [hunk 1]
```diff
@@ -1570,6 +1578,14 @@ dependencies = [
...
 ]
 
+[[package]]
+name = "quote"
+version = "1.0.2"
+source = "registry+https://github.com/rust-lang/crates.io-index"
+dependencies = [
+ "proc-macro2 1.0.6 (registry+https://github.com/rust-lang/crates.io-index)",
+]
+
```
Comment (jyn514) [inline Cargo.lock]: I don't like that we're building yet another version of syn/quote, but I don't see a good way to fix this without upgrading all our  dependencies.

File: Cargo.lock [hunk 2]
```diff
@@ -2574,7 +2600,15 @@ name = "toml"
 version = "0.4.8"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 dependencies = [
- "serde 1.0.84 (registry+https://github.com/rust-lang/crates.io-index)",
+ "serde 1.0.104 (registry+https://github.com/rust-lang/crates.io-index)",
+]
+
+[[package]]
+name = "toml"
+version = "0.5.5"
```
Comment (jyn514) [inline Cargo.lock]: While you're upgrading, can you bump `toml` to 0.5 so it's not duplicated?

Comment (pietroalbini) [reply Cargo.lock]: Done.

**Comment by pietroalbini** [comment]
Resolved all the review comments.

**Comment by jyn514** [comment]
Looks great, thanks!

**Pull request merged**